src/System.Numerics.Vectors/src/System/Numerics/Vector.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Diagnostics.Contracts;
```

Comment:
This was caused by a previous change being applied only to the .cs file and not to the .tt file. It's not related to your stuff, but can you remove this line from the .tt file as well?